src/api/v4/native/utils.js

```diff
@@ -0,0 +1,26 @@
+
+function isLeafletLoaded () {
```

Comment:
could move this to `core/util` ?

Reply:
It could be, but it is in the API scope so I think a good place is in the `api` directory

Reply:
I'm for that too. We are not exposing these functions in the public API so moving it to the internals so we can use it throughout the whole codebase.

Reply:
Not a big deal so go for it :)